src/modules/webhooks/send-notification/1.0/module.json

```diff
@@ -0,0 +1,228 @@
+{
+    "api": {
+      "url": "https://dev-api.edifydev.net/v4/notifications",
+      "method": "post",
+      "header": {
+        "u": "{{edify.user.email}}:{{edify.user.apiToken}}",
+        "content-type": "application/json"
+      },
+      "onResponse": {
+        "valueAccessor": "",
+        "valueMap": {
+          "any": {
+            "label": "",
+            "actions": [],
+            "exitPort": "",
+            "exitPortValueMap": {},
+            "exitPortValueAccessor": "",
+            "httpCode": "",
+            "collapsed": false
+          }
+        }
+      }
+    },
+    "name": "Send Notification",
+    "sections": [
+      {
+        "label": "Notification",
+        "inputs": [
+          {
+            "name": "type",
+            "type": "dropdown",
+            "label": "Type",
+            "values": [
+              {
+                "label": "HTML5",
+                "value": "html5"
+              },
+              {
+                "label": "Push Notification",
+                "value": "push"
```

Comment:
Just got word from Matt to not do push notifications actually, since mobile apps won't handle them well. So we can remove any other fields that were for push notifications